src/components/ImportFile/ImportFileContent/ImportFileAccept.js

```diff
@@ -0,0 +1,58 @@
+import ClassNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '../../Icon';
+import { ProgressBar } from '../../ProgressBar';
+
+const ImportFileAccept = ({
+  progressDone,
+  acceptedFiles,
+  rejectedFiles,
+  text,
+  icon,
+  className,
+  ...props
+}) => {
+  const classes = ClassNames('import-file-accept-text-pf', className);
```

Comment:
this needs to be `classNames(` it was decided to use `import classNames from 'classnames` to keep code consistent.  An elint rule is coming soon.
https://github.com/patternfly/patternfly-react/blob/master/CONTRIBUTING.md#code-consistency